_includes/global_fields.md

```diff
@@ -4,6 +4,7 @@ A set of common fields that appear on all resources is included first, for refer
 
 | Name          | Type      | Description
 |-----------    |-----------|-----------|--------------
+|id 			| string    | A string representing a local identifier for the resource on the local system.  It may be a string, integer, guid, etc.  This value MUST be suitable for use in a helper function such as add_tags, add_lists to identify the corresponding resource
```

Comment:
The other con is now we're explicitly saying something about URL conventions, which we've previously largely avoided. But I'd say if we go this way (rather than the array way Brian is working on) we kind of might as well, since it's a boon for developers to be able to use this ID to construct URLs, and it's kind of what they expect.